output_with_summary.tmpl

```diff
@@ -1,24 +1,27 @@
+{{- if not .Matches }}
+No vulnerabilities found.
+{{- else }}
 Vulnerabilities:
-{{- $negligible := 0 }}
-{{- $low := 0 }}
-{{- $medium := 0 }}
-{{- $high := 0 }}
-{{- $critical := 0 }}
-{{- $total := 0 }}
-{{- range .Matches}}
-    {{ .Artifact.Name }} (type {{ .Artifact.Type }}) version {{ .Artifact.Version }}: {{ .Vulnerability.ID }} severity {{ .Vulnerability.Severity }}, link: {{ .Vulnerability.DataSource }}
-    {{- if eq .Vulnerability.Severity "Negligible" }}
-        {{- $negligible = add1 $negligible }}
-    {{- else if eq .Vulnerability.Severity "Low"}}
-        {{- $medium = add1 $low }}
-    {{- else if eq .Vulnerability.Severity "Medium"}}
-        {{- $medium = add1 $medium }}
-    {{- else if eq .Vulnerability.Severity "High"}}
-        {{- $medium = add1 $high }}
-    {{- else if eq .Vulnerability.Severity "Critical"}}
-        {{- $medium = add1 $critical }}
-    {{- end }}
-    {{- $total = add1 $total }}
-{{- end}}
-
+    {{- $negligible := 0 }}
+    {{- $low := 0 }}
+    {{- $medium := 0 }}
+    {{- $high := 0 }}
+    {{- $critical := 0 }}
+    {{- $total := 0 }}
+    {{- range .Matches}}
+        {{ .Artifact.Name }} (type {{ .Artifact.Type }}) version {{ .Artifact.Version }}: {{ .Vulnerability.ID }} severity {{ .Vulnerability.Severity }}, link: {{ .Vulnerability.DataSource }}
```

Comment:
I am wondering this should might look better with `package: xxx | version: xxx | severity: xxx | link: xxx`